runtime/calamari/Cargo.toml

```diff
@@ -17,6 +17,7 @@ smallvec = "1.6.1"
 
 # Substrate primitives
 sp-api = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.15" }
+sp-arithmetic      = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.15" }
```

Comment:
Formatting

Reply:
it's kinda strange how `cargo fmt` is executed autmatically on file save in code, but somehow, sometimes it just finishes without actually reformatting.

Reply:
Yeah also not sure why this is not failing the CI fmt check...

Reply:
is resolved